index.js

```diff
@@ -18,6 +18,17 @@ const Funnel = require('broccoli-funnel');
 const p = require('ember-cli-preprocess-registry/preprocessors');
 const existsSync = require('exists-sync');
 
+// Backport of _findHost for ember-cli < 2.7
+// See https://github.com/ember-cli/ember-cli/blob/0d17e839ca7a5128d1b95b9981a7e77ed0e55ead/lib/models/addon.js#L615
+function findHostShim() {
```

Comment:
Don't know. Only thing I found mentioned was this: `It is also preferable that your app is running ember-cli 2.12.0 and higher.` If this is a requirement it should be stated so...

Other problem with skipping this is that we are relying on a private API here (`_findHost`). Maybe this use case is a good reason to revisit https://github.com/ember-cli/ember-cli/pull/6755?